recipes/cinemagoerng/conda_build_config.yaml

```diff
@@ -0,0 +1 @@
+python_min: '3.10'
```

Comment:
I prefer to have this defined in the recipe itself, making it easier to remove/update when the time comes.